src/app/pages/system/general-settings/support/sys-info/sys-info.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator/jest';
+import { LicenseInfoInSupport } from 'app/pages/system/general-settings/support/license-info-in-support.interface';
+import { SysInfoComponent } from 'app/pages/system/general-settings/support/sys-info/sys-info.component';
+
+describe('SysInfoComponent', () => {
+  const systemInfo = {
+    version: 'TrueNAS-SCALE-22.12-MASTER-20220318-020017',
+    system_product: 'VirtualBox',
+    model: 'AMD Ryzen 3 3200G',
+    memory: '5 GiB',
+    serial: 'ffbb355c',
+  };
+  const licenseInfo = {
+    customer_name: 'iXsystems Inc.',
+    features: ['DEDUP', 'FIBRECHANNEL', 'VM'],
+    contract_type: 'GOLD',
+    expiration_date: '2022-06-10',
+    daysLeftinContract: -4,
+    add_hardware: 'NONE',
+  };
+
+  let spectator: SpectatorHost<SysInfoComponent>;
+  const createHost = createHostFactory({
+    component: SysInfoComponent,
+  });
+
+  beforeEach(() => {
+    spectator = createHost('<ix-sys-info [system_info]="systemInfo"></ix-sys-info>', {
+      hostProps: {
+        systemInfo,
+        has_license: false,
+      },
+    });
+  });
+
+  it('it shows a block with system info', () => {
+    const sysInfoItems = spectator.queryAll('.sys-info-wrapper .value');
```

Comment:
would be good to also test labels. You could use reduce to form a dictionary: { Label1: Value, Label2: Value }, etc.